app/containers/Settings/Settings.jsx

```diff
@@ -116,141 +120,87 @@ export default class Settings extends Component<Props, State> {
-                  <div className="walletKey">{account.label}</div>
-                  <div
-                    className="deleteWallet"
-                    onClick={() =>
-                      this.deleteWalletAccount(account.label, account.key)
-                    }
-                  >
-                    <Delete />
-                  </div>
-                </div>
-              </div>
-            ))}
-          </div>
-          <Button onClick={() => this.saveWalletRecovery()}>
-            Export wallet recovery file
-          </Button>
-          <Button onClick={this.loadWalletRecovery}>
-            Load wallet recovery file
-          </Button>
-        </div>
-      </div>
+          <Tooltip title="Add Token" className={styles.headerButtonContainer}>
+            <AddIcon className={styles.add} />
+            <span>Add Token</span>
+          </Tooltip>
+        </UnderlinedHeader>
+        <Panel
+          className={styles.settingsPanel}
+          renderHeader={this.renderHeader}
+        >
+          <section className={styles.settingsItemsContainer}>
+            <SettingsItem title="THEME">
+              <SelectInput
+                items={Object.keys(themes)}
+                value={theme}
+                onChange={this.updateThemeSettings}
+              />
+            </SettingsItem>
+            <SettingsItem title="CURRENCY">
+              <SelectInput
+                items={Object.keys(CURRENCIES).map(value =>
+                  value.toUpperCase()
+                )}
+                value={currency.toUpperCase()}
+                onChange={this.updateCurrencySettings}
+                getItemValue={value => value.toLowerCase()}
+              />
+            </SettingsItem>
+            <SettingsItem title="BLOCK EXPLORER">
+              <SelectInput
+                items={Object.keys(EXPLORERS).map(value => EXPLORERS[value])}
+                value={explorer}
+                onChange={this.updateExplorerSettings}
+              />
+            </SettingsItem>
+            <SettingsLink to={ROUTES.ENCRYPT} title="ENCRYPT A KEY" />
+            <SettingsLink to={ROUTES.NODE_SELECT} title="NODE SELECTON" />
+            <WalletRecoveryPanel
+              title="WALLET RECOVERY"
+              loadWalletRecovery={this.loadWalletRecovery}
+              saveWalletRecovery={this.saveWalletRecovery}
+            />
+            <div className={styles.settingsDonations}>
+              Created by CoZ. Donations: Adr3XjZ5QDzVJrWvzmsTTchpLRRGSzgS5A
+            </div>
+          </section>
+        </Panel>
+      </section>
     )
   }
+
+  renderHeader = () => (
+    <div className={styles.settingsPanelHeader}>
+      <div className={styles.settingsPanelHeaderItem}>
+        Manage your neon wallet
+      </div>
+      <div className={styles.settingsPanelHeaderItem}>
+        Community Support: https://discordapp.com/invite/R8v48YA
```

Comment:
Lets make this actually open a browser

Reply:
do you want to change the styling also so it looks clickable? underlined on hover?